cmd/upload.go

```diff
@@ -32,6 +33,13 @@ func init() {
 				progressBar := progressbar.New(len(args) + 1)
 				progressBar.RenderBlank()
 
+				// TODO
+				// In the future, perhaps we do something a little more graceful than to
+				// use the git metadata from the last file uploaded. It seems we
+				// shouldn't ever be uploading files from multiple repositories under a
+				// single mapset.
```

Comment:
Would it be difficult to add a sanity check so it'd error out if they were different?

Reply:
It wouldn't be, though with our plans to replace files as an argument array with an individual directory this will go away. I'm fine with allowing this for now given it's a strange edge case anyway.